### DISCUSSION TITLE
"Planning to play" list

### DISCUSSION THREAD

**Discussion by VoiceOfAutumn** [RetroAchievements/RAWeb#799] [Ideas]
Is it possible to have a similar function like "Planning to Watch" on platforms like Netflix or lists like Anilist. I often run into sets i'd like to play later on, but I can't easily click a "+" to add them to some sort of list in-site.

**Comment by Jamiras** [comment]
See also #731. "Reminder" could just be an additional status like "Favorite".

**Comment by Jamiras** [comment]
This was added in 4.0.0 by #1815.
